recipe/meta.yaml

```
@@ -508,7 +533,7 @@ source:
     md5: {{ cudavars[major_minor]["checksums"]["win"] }}  # [win]
 
 build:
-  number: 10
+  number: 0
```

Would leave this as `10`. Since we build all versions of this package in this feedstock, some of them will need the build number bump due to previous changes

```suggestion
  number: 10
```

---

cc @leofang

---

We are not just building 11.8?!

---

No please look at the CI jobs here

---

Normally we won't get uploads unless something changed with the other package versions. In this case, as we have [moved to `.conda` packages]( https://conda-forge.org/docs/user/announcements.html#moving-to-conda-artifacts ), there will be new packages being produced for all versions.